test/CMakeLists.txt

```diff
@@ -89,6 +90,9 @@ target_include_directories(gtest_zlib PRIVATE
     ${CMAKE_SOURCE_DIR}
     ${CMAKE_BINARY_DIR})
 
+find_package(Threads REQUIRED)
```

Comment:
It would be good to include `test_deflate_concurrency.cc` only if `Threads` is found.

Reply:
Done.